deploy/osd-logging/03-storage-quota.yaml

```
@@ -0,0 +1,8 @@
+- apiVersion: v1
+  kind: ResourceQuota
+  metadata:
+    name: logging-storage-quota
+    namespace: openshift-logging
+  spec:
+    hard:
+      requests.storage: "600G"
```

```suggestion
      requests.storage: "600Gi"
```

I believe the unit is `Gi`, not `G`?